docs/cli/custom-objects-exporter.md

```diff
@@ -0,0 +1,87 @@
+# Custom Objects Exporter
+
+A package that helps with exporting [commercetools custom objects](https://docs.commercetools.com/http-api-projects-custom-objects.html) in `JSON` format from the [commercetools platform](https://docs.commercetools.com/).
+
+## Configuration
+
+The constructor accepts two arguments:
+
+* A required object containing the following values:
+  * `apiConfig` (Object): `AuthMiddleware` options for authentication on the commercetools platform. (Required. See [here](https://commercetools.github.io/nodejs/sdk/api/sdkMiddlewareAuth.html#named-arguments-options))
+  * `accessToken` (String): Access token to be used to authenticate requests to API. Requires scope of [`view_products`], [`view_orders`], [`view_customers`]
```

Comment:
maybe a link to the package or docs that helps dev to retrieve their tokens?

Reply:
why in brackets for `view_products`, btw?

Reply:
Good idea, will add that. 
Regarding the brackets, I saw other packages in the monorepo used brackets so I wanted to keep it consistent.

Reply:
[`view_products`], [`view_orders`], [`view_customers`] -> [`view_products`, `view_orders`, `view_customers`]